### PR TITLE
Add bootstrap page module to server modules

### PR DESCRIPTION
**Pull request by fstanis**

### CODE REVIEW

File: example/package.json [hunk 1]
```diff
@@ -25,5 +25,6 @@
     "files": [
       "test/*.test.js"
     ]
-  }
+  },
+  "browserslist": ["last 1 Chrome versions"]
```
Comment (alabiaga) [inline example/package.json]: This is only for tests at the moment? Do we want to include other browsers in the future?

File: example/server-modules/bootstrapPage.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    document.write(event.data.amp);
+    document.close();
+  }
+}
+window.addEventListener('message', message);`;
+
+const BOOTSTRAP_HTML = generateHTMLFromJS(BOOTSTRAP_JS);
+
+const CSP_DIRECTIVES = [
+  "default-src 'none'",
```
[the remark this answers is not in the document]
Comment (fstanis) [reply example/server-modules/bootstrapPage.js]: Good point, done.

File: example/package.json (hunk 1 above)
Comment (fstanis) [reply example/package.json]: Yes, the target for this build is mainly puppeteer. I added a comment to explain this. Without this, Parcel bundles it in a way which requires `regeneratorRuntime`. Context: https://github.com/parcel-bundler/parcel/issues/1762#issuecomment-504438377

File: example/server-modules/bootstrapPage.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
+const path = require('path');
+const crypto = require('crypto');
+
+const HOST_ORIGIN = 'http://localhost:3000';
```
[the remark this answers is not in the document]
Comment (fstanis) [reply example/server-modules/bootstrapPage.js]: This is tricky. I think it's bad practice for CSP headers to be too automatic / configurable. On the other hand, this is causing me some headache right now with tests. Suggestions welcome on what a good compromise is (not encouraging people to autogenerate CSP vs usability).

[the remark this answers is not in the document]
Comment (alabiaga) [reply example/server-modules/bootstrapPage.js]: This is fine. If in use, port will be reassigned anyway.

**Pull request merged**